logger.js

```diff
@@ -42,7 +42,7 @@ function pinoLogger (opts, stream) {
 
   function getValidLogLevel (level, defaultValue = 'info') {
     if (level && typeof level === 'string') {
-      const logLevel = level.trim().toLowerCase()
```

Comment:
The purpose here is to compare names in a consistent manner.

Reply:
Hi @jsumners, thanks for the response! 

**My reasoning for the change:** 

1. `validLogLevels` contains values that are not mapped to lowercased values, so the check will always fail for non-lowercased values. (like `infoCustom` example above). 
2. We could potentially prepare the `validLogLevels` by mapping each value to lowercase, so the check will result to `true`. After that, the `getValidLogLevel` will return a lowercased value, and the code of pino-http will try calling something like `logger.infocustom` which will also throw (because the valid call would be `logger.infoCustom`)

Pls let me know what you think, thanks